python/dgl/graphbolt/minibatch.py

```diff
@@ -500,6 +500,49 @@ def apply_to(x, device):
         return self
 
 
+    def to_pyg_adapter(self, device):
```

Comment:
Why is there a need to pass the device parameter here?

Reply:
We can just keep the data on the same device it currently is on.

Reply:
solved it